docs/swagger.yml

```diff
@@ -2372,22 +2372,32 @@ definitions:
           type: "string"
           description: "The insights client id"
           example: "b077190b74f14b3582e57ce8243ae356"
-        system_intent_role:
+        system_purpose:
```

Comment:
https://github.com/quipucords/quipucords/blob/31c306cd82dbc067c7f1378ea4b6e8786ce02f74/docs/swagger.yml#L1380 

Should we also add it to the Host definition in the swagger?

Reply:
Done